pkg/apis/api/types.go

```diff
@@ -714,9 +714,10 @@ type TestOutput struct {
 }
 
 type Releases struct {
-	Releases    []string             `json:"releases"`
-	GADates     map[string]time.Time `json:"ga_dates"`
-	LastUpdated time.Time            `json:"last_updated"`
+	Releases        []string             `json:"releases"`
+	GADates         map[string]time.Time `json:"ga_dates"`
+	DevelStartDates map[string]time.Time `json:"devel_start_dates"`
+	LastUpdated     time.Time            `json:"last_updated"`
```

Comment:
I feel like if we're tracking a second set of dates, we might someday add a third, or fourth.   What if we want to track support end dates, for example?

Also s/Devel/Development/.  The extra characters saved I don't think are offset by the reduction in clarity.

WDYT?

```golang
type ReleaseDates struct {
	GA               *time.Time `json:"ga,omitempty"`
	DevelopmentStart *time.Time `json:"development_start,omitempty"`
}

type Releases struct {
	Releases          []string                 `json:"releases"`
	Dates             map[string]ReleaseDates  `json:"dates"`
	DeprecatedGADates map[string]time.Time     `json:"ga_dates"` // for backward compatibility
	LastUpdated       time.Time                `json:"last_updated"`
}
```